swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=0,fabric="Unknown"):
+        super().__init__(id, condition)
+        self.fabric = fabric if fabric else "Unknown"
```

Comment:
The one small thing to note here is that you have already set the default for fabric to "Unknown". Remember that strings are immutable, so we don't need to treat them the same way we might treat a mutable default value like a list. The conditional here is unnecessary!